supplementary_style_guide/glossary_terms_conventions/general_conventions/b.adoc

```diff
@@ -60,6 +60,16 @@
 
 *See also*: xref:bare-metal-n[bare metal (noun)]
 
+[[bare-metal-service]]
+==== image:images/yes.png[yes] Bare Metal service (noun)
+*Description*: The Bare Metal service is a Red Hat OpenStack Platform product that provisions bare metal machines.
```

Comment:
```suggestion
*Description*: The _Bare Metal service_ is a Red Hat OpenStack Platform product that provisions bare metal machines.
```
Per ISG: Words and phrases that are being defined or explained should use italic text: https://www.ibm.com/docs/en/ibm-style?topic=format-highlighting.